api/optimeet/groups/models.py

```diff
@@ -32,3 +32,8 @@ class Recommendations(models.Model):
     place_url = models.CharField(max_length=100)
     times = models.JSONField()
     
+class Votes(models.Model):
+    group_id = models.ForeignKey(Group, related_name='group_votes', on_delete=models.CASCADE)
+    user_id = models.CharField(max_length=64, blank=True)
+    results_of_voted = models.CharField(max_length=64, blank=True)
```

Comment:
How would `results_of_voted` be used? Since this is a single vote that a particular user is making in a group we wouldn't keep track of multiple votes. Also remember what we're voting for: recommendations, so we should add the recommendation id this vote is for as a foreign key.